README.md

```diff
@@ -50,6 +50,12 @@ You can shell into the machine with `vagrant ssh` or `ssh -p 2222 vagrant@localh
 * (optional) to enable fedora internal audit capability, edit `install_scripts/config` and change the FEDORA_AUDIT variable to true. The FEDORA_AUDIT_LOCATION can also be changed from its default "/audit", if necessary.
 * `vagrant up`
 
+###Acceptance Testing
+
+One may validate the state of the environment provisioned by the BASH scripts using [beaker](https://github.com/puppetlabs/beaker).
+The acceptance tests (implemented using [server-spec](http://serverspec.org/)) may be executed locally in the following manner:
+* `bundle exec rspec spec/acceptance`
```

Comment:
@awoods:
> Would it be possible to design the acceptance tests executed in this bundle command to run within the VirtualBox itself? Would that make sense?

Yes, these tests can be integrated into a `rake` Task, which can also address Gem dependency resolution using the Bundler.

> Additionally, pushing these files ("GemFile", "spec/*") into a "test" directory, or the like, would help make it clear which files are needed for the VM and which are extra/optional.

Certainly, I can work to address these two issues before the Fedora Tech. Call scheduled for 12/24/15 (or any alternative date).